website/content/en/preview/concepts/metrics.md

```diff
@@ -102,3 +102,8 @@ Size of the request batch per batcher
 ### `karpenter_cloudprovider_batcher_batch_time_seconds`
 Duration of the batching window per batcher
 
+## Pricing Metrics
+
+### `karpenter_pricing_instance_estimate`
```

Comment:
```suggestion
### `karpenter_cloudprovider_instance_price`
```

Can we change the naming a bit? I was hoping that we could create a whole category of metrics that exist in the same namespace. Imagine:

1. `karpenter_cloudprovider_instance_price`
2. `karpenter_cloudprovider_instance_available`
3. `karpenter_cloudprovider_instance_vcpus`
4. `karpenter_cloudprovider_instance_memory`
5. ...